fourkitchens/suhumsci
# SHS-4969: adjust size of field autocomplete suggestions

# READY FOR REVIEW

## Summary
- Override autocomplete sizing function to ensure that the width of suggestions container will be the width of the text field 
- Adjust height and scroll behavior of field autocomplete suggestions container

## Urgency
- normal

## Steps to Test
1. Create a Person node. Go to the "Bio Information" tab and start typing in the URL field of the "Info Links" section. Make sure that the autocomplete suggestions width is always the width of the text field. Also check that the height is limited to a max of 500px and that the list is scrollable in case there are many results.
2. Create a Flexible page and add a "Hero (Layered) - Slider" paragraph. Scroll to the link field and repeat the tests in the previous step. In this case, the linkit widget is used instead of the default drupal link field, but the behavior should be the same.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
